src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/MappingHelpers.kt

```diff
@@ -0,0 +1,32 @@
+package org.vaccineimpact.api.app.repositories.jooq
+
+import org.jooq.Record
+import org.vaccineimpact.api.app.errors.BadDatabaseConstant
+import org.vaccineimpact.api.db.Tables.BURDEN_ESTIMATE_SET
+import org.vaccineimpact.api.db.Tables.BURDEN_ESTIMATE_SET_TYPE
+import org.vaccineimpact.api.models.BurdenEstimateSetType
+import org.vaccineimpact.api.serialization.Deserializer
+import org.vaccineimpact.api.serialization.UnknownEnumValue
+
+class MappingHelpers(val deserializer: Deserializer = Deserializer())
+{
+    inline fun <reified T : Enum<T>> mapEnum(raw: String): T
+    {
+        return try
+        {
+            deserializer.parseEnum(raw)
+        }
+        catch (e: UnknownEnumValue)
+        {
+            throw BadDatabaseConstant(e.name, e.type)
+        }
+    }
+
+    fun mapBurdenEstimateSetType(record: Record) =
+            mapBurdenEstimateSetType(
```

Comment:
I like the move of mapping logic into its own class, as the repos are getting very logic heavy as we've discussed before. I'd be pro moving similar mapping logic for other types out of repos. Bu these methods are v specific to burden estimates and I think if we go with this approach we should have separate mapping logic classes, which individual repos can use as needed, rather than one giant mapping helper class. What do you think?

Reply:
I've made changes as you suggested.